### PR TITLE
category name changed on all slides after ajax success

### PR DESCRIPTION
**Pull request by manthanchauhan**

### CODE REVIEW

File: online_portfolio/static/portfolio/js/edit.js [hunk 1]
```diff
@@ -708,7 +714,16 @@ function updateCategoryName(element, slideIndex) {
         data: {"oldName": oldName, "newName": newName},
         dataType: "json",
         success: function (data){
-            // add name change logic for frontend.
+            const headings =  document.querySelectorAll(".sectionHeadingDiv");
+            headings.forEach(heading=>{
+                const headingCategory = heading.getAttribute('category');
+                if(headingCategory === oldName){
+                    heading.setAttribute('category',newName);
+                    heading.querySelector("h2").innerHTML=newName;
```
Comment (manthanchauhan) [inline online_portfolio/static/portfolio/js/edit.js:722]: @HarshitaAgarwal09 this is fine, but I was thinking is there a better way? I mean if we in the future accidentally add another `<h2>` element in the `sectionHeadingDiv` this logic will break. It's not maintainable.

Comment (manthanchauhan) [reply online_portfolio/static/portfolio/js/edit.js:722]: Also, we still missed changing the category attribute in lines 542, 547, and 545. I have no idea how come I not mention those. Basically after deletion `oldName` should not exist in the frontend, and should be replaced by `newName` be it an attribute or value or text.

Comment (manthanchauhan) [reply online_portfolio/static/portfolio/js/edit.js:722]: I'd advise you to check that skill section has no trace of `oldName` once deleted.

Comment (HarshitaAgarwal09) [reply online_portfolio/static/portfolio/js/edit.js:722]: Okay

Comment (HarshitaAgarwal09) [reply online_portfolio/static/portfolio/js/edit.js:722]: @manthanchauhan  I am unable to access the category property from the tags as it is the custom attribute.

Comment (HarshitaAgarwal09) [reply online_portfolio/static/portfolio/js/edit.js:725]: @manthanchauhan Updated the code. Please check it

Comment (HarshitaAgarwal09) [reply online_portfolio/static/portfolio/js/edit.js:725]: @ there is no other way to acces the element h2 without accessing it this way. Also updated the code ,Please check.

File: online_portfolio/static/portfolio/js/edit.js [hunk 2]
```diff
@@ -708,7 +717,37 @@ function updateCategoryName(element, slideIndex) {
...
-            // add name change logic for frontend.
+            const headings =  document.querySelectorAll(".sectionHeadingDiv");
+            headings.forEach(heading=>{
+                const headingCategory = heading.getAttribute('category');
+                if(headingCategory === oldName){
+                    heading.setAttribute('category',newName);
+                    
+                    const headingText = heading.querySelector("h2");
+                    headingText.innerHTML = newName;
+                    headingText.setAttribute('category', newName);
```
[the remark this answers is not in the document]
Comment (manthanchauhan) [reply online_portfolio/static/portfolio/js/edit.js]: Solved it!! But be careful, I can't always look for each and every change you make.

**Pull request merged**